SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -292,30 +292,35 @@ int64 USpatialActorChannel::ReplicateActor()
 
 	ActorReplicator->RepState->LastCompareIndex = ChangelistState->CompareIndex;
 
+	FClassInfo* Info = NetDriver->TypebindingManager->FindClassInfoByClass(Actor->GetClass());
+
 	// Update the handover property change list.
-	FHandoverChangeState HandoverChangeState = GetHandoverChangeList(*ActorHandoverShadowData, Actor);
+	if (ActorHandoverShadowData != nullptr)
```

Comment:
We create this in `OnReserveEntityIdResponse`, and that would've had to happen before getting to this point, so I'd say
`checkSlow(ActorHandoverShadowData)` should be enough?